src/SimpleExcelReader.php

```diff
@@ -154,6 +161,31 @@ public function getRows(): LazyCollection
         });
     }
 
+    public function getHeaders(): ?array
+    {
+        if ($this->processHeader && ! $this->headers) {
```

Comment:
There's a big piece of indented code here. I can be simplified by first checking if headers is set to something. If it is, we can already return the headers.